pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -658,6 +678,19 @@ public void run() {
     }
   }
 
+  private void checkFastSyncOptions() throws ParameterException {
+    if (fastSyncMaxWaitTime < 0) {
+      throw new ParameterException(
+          commandLine, "--fast-sync-max-wait-time must be a positive or zero integer");
+    }
+    checkOptionDependencies(
+        logger,
+        commandLine,
+        "--sync-mode",
+        syncMode.equals(SyncMode.FAST),
```

Comment:
I personally prefer the reverse pattern to prevent NPEs, even if here I don't see a way for it to be null, but imagine PicoCLI has something strange and injects it null in some cases. And I think this is a good habit to do it all the time in case of.
```suggestion
        SyncMode.FAST.equals(syncMode),
```